src/Reddit/Reddit.ts

```diff
@@ -1,427 +1,644 @@
+
+    for (el of items.reverse()) {
+      if (PluginBase.util.isVisible(el)) {
+        if (isOldReddit) {
+          return select<HTMLElement>(comExpBtn, el)!.click()
+        } else {
+          return clickIfDisplayed(el.parentNode as HTMLElement)
+        }
+      }
+    }
+  }
+}
+
+async function expandAll() {
+  const { comment, comExpBtn } = composeExpandableElementsSelectors()
+  const selector = isOldReddit ? `${ comment } ${ comExpBtn }` : comExpBtn
+
+  for (let el of selectAll<HTMLElement>(selector)) {
+    if (isOldReddit) {
+      el.click()
+    } else {
+      clickIfDisplayed(el.parentNode as HTMLElement)
+    }
+  }
+}
+
+function collapseCurrent() {
+  const { postExpBtn, comExpBtn } = composeCollapseBtnSelector()
+
+  const postBtn = !!postExpBtn && select<HTMLElement>(postExpBtn!) || null
+  const commentBtns = selectAll<HTMLElement>(comExpBtn)
+
+  postBtn && PluginBase.util.isVisible(postBtn!) && postBtn!.click()
+
+  for (const el of commentBtns) {
+    if (PluginBase.util.isVisible(el)) {
+      el.click()
+      break
+    }
+  }
+}
+
+function toggleContext(isPostContext = false) {
+  if (isPostContext) {
+    PluginBase.util.prependContext('Post')
+    PluginBase.util.removeContext('Post List')
+  } else {
+    PluginBase.util.prependContext('Post List')
+    PluginBase.util.removeContext('Post')
+  }
+}
+
+export default <IPluginBase & IPlugin> {
   ...PluginBase,
   ...{
-    niceName: "Reddit",
-    description: "Commands for Reddit.com",
-    version: "4.4.0",
+    niceName: 'Reddit',
+    description: 'Commands for Reddit.com',
+    version: '4.4.0',
     apiVersion: 2,
     match: /^https?:\/\/.*\.reddit.com/,
-    authors: "Miko",
+    authors: 'Miko',
 
     // less common -> common
     homophones: {
-      navigate: "go",
-      contract: "collapse",
-      claps: "collapse",
-      expense: "expand",
-      explain: "expand",
-      expanding: "expand",
-      "expand noun": "expand 9",
-      "it's been": "expand",
-      expanse: "expand",
-      expanded: "expand",
-      stand: "expand",
-      xpand: "expand",
-      xmen: "expand",
-      spend: "expand",
-      span: "expand",
-      spell: "expand",
-      spent: "expand",
-      "reddit dot com": "reddit",
-      "read it": "reddit",
-      shrink: "collapse",
-      advert: "upvote",
-      download: "downvote",
+      navigate: 'go',
+      contract: 'collapse',
+      claps: 'collapse',
+      expense: 'expand',
+      explain: 'expand',
+      expanding: 'expand',
+      'expand noun': 'expand 9',
+      'it\'s been': 'expand',
+      expanse: 'expand',
+      expanded: 'expand',
+      stand: 'expand',
+      xpand: 'expand',
+      xmen: 'expand',
+      spend: 'expand',
+      span: 'expand',
+      spell: 'expand',
+      spent: 'expand',
+      'reddit dot com': 'reddit',
+      'read it': 'reddit',
+      shrink: 'collapse',
+      advert: 'upvote',
+      download: 'downvote',
+      commence: 'comments',
+      what: 'upvote',
```

Comment:
please remove "what" it is too common of a word to be a homophone and will cause issues